src/PNE/pne_ecrire_jeu_de_donnees_mps.c

```diff
@@ -25,13 +25,22 @@
 # include "pne_sys.h"
 # include "pne_define.h"
 # include "pne_fonctions.h"
-
 # ifdef PNE_UTILISER_LES_OUTILS_DE_GESTION_MEMOIRE_PROPRIETAIRE	
   # include "pne_memoire.h"
 # endif
-
+#define SRS_BUFFER_SIZE 1024
 /*----------------------------------------------------------------------------*/
-
+void MettreAjourNom(int NomParDEfaut, char* Nom,  const char* NouveauNom, int Numero, char Type){
+  memset(Nom, 0, SRS_BUFFER_SIZE);
+  if (NomParDEfaut)
```

Comment:
```suggestion
void MettreAjourNom(int NomParDefaut, char* Nom,  const char* NouveauNom, int Numero, char Type){
  memset(Nom, 0, SRS_BUFFER_SIZE);
  if (NomParDefaut)
```

Reply:
I'm not sure `memset` is necessary here. Memory will be overwritten by a null-terminated string afterwards, so why bother ?

I may be wrong here.

Reply:
you're right